hashicorp/boundary-ui
# Alias create

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/JIRA_TICKET_NUMBER)

## Description



## 

https://github.com/hashicorp/boundary-ui/assets/15043878/f79b48d4-ad85-4a5d-8396-39c9decf1608

Screenshots (if appropriate):

## How to Test

mirage: 
- click on aliases on left side bar and create a new alias and it should successfully save
- there is no validation here since this is mirage 

controller:
- get the latest main from BE, make install and run the controller
- you should be able to create aliases by finding the link on the left sidebar
- click on the new alias creation 
- enter `hostid, target id, name, desc, alias value` 
- hit save
- to view validation: submit the form without alias value as that is the only required field





:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:
